src/components/SQForm/useForm.js

```diff
@@ -33,11 +29,11 @@ function _getHasValue(meta) {
     return !!fieldValue.length;
   }
 
-  if (fieldValue) {
+  if (typeof fieldValue === 'number') {
     return true;
   }
 
-  return false;
+  return !!fieldValue ?? false;
```

Comment:
this got me thinking... what does `!!fieldValue` eval to when `fieldValue` is undefined or null? False! This is minor, and not a blocker per se, but... isn't the nullish coalescing unnecessary here?